bundles/io/org.openhab.io.multimedia.tts.googletts/pom.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns="http://maven.apache.org/POM/4.0.0"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+
+	<parent>
+		<groupId>org.openhab.bundles</groupId>
+		<artifactId>io</artifactId>
+		<version>1.7.0-SNAPSHOT</version>
+	</parent>
+
+	<name>openHAB Multimedia GoogleTTS</name>
+
+	<properties>
+		<bundle.symbolicName>org.openhab.io.multimedia.tts.googletts</bundle.symbolicName>
+		<bundle.namespace>org.openhab.io.multimedia.tts.googletts</bundle.namespace>
+		<project.dependencyDirectory>${basedir}/resources</project.dependencyDirectory>
+		<deb.name>openhab-addon-io-multimedia-googletts</deb.name>
+		<deb.description>${project.name}</deb.description>
+	</properties>
+
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.openhab.io</groupId>
+	<artifactId>org.openhab.io.multimedia.tts.googletts</artifactId>
+
+	<packaging>eclipse-plugin</packaging>
+
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>com.savage7.maven.plugins</groupId>
+				<artifactId>maven-external-dependency-plugin</artifactId>
+				<configuration>
+					<stagingDirectory>
+						${basedir}/lib
+					</stagingDirectory>
+					<artifactItems>
+
+					</artifactItems>
+				</configuration>
+				<executions>
+					<execution>
+						<id>resolve-install-external-dependencies</id>
+						<phase>process-resources</phase>
+						<goals>
+							<!-- mvn com.savage7.maven.plugins:maven-external-dependency-plugin:resolve-external -->
+							<goal>resolve-external</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>org.vafer</groupId>
+				<artifactId>jdeb</artifactId>
+			</plugin>
+		</plugins>
+
+		<pluginManagement>
+			<plugins>
+				<plugin>
+					<groupId>com.savage7.maven.plugins</groupId>
+					<artifactId>maven-external-dependency-plugin</artifactId>
+					<version>0.4</version>
+				</plugin>
+			</plugins>
+		</pluginManagement>
```

Comment:
Sorry, copied the pom from another bundle and didn't remove this unnecessary plugin.